core/templates/dev/head/pages/exploration_editor/StateEditorTutorialFirstTimeService.js

```diff
@@ -24,14 +24,16 @@ oppia.factory('StateEditorTutorialFirstTimeService', [
 
     var STARTED_TUTORIAL_EVENT_URL = '/createhandler/started_tutorial_event';
 
+    var canEdit = GLOBALS.can_edit;
```

Comment:
Can we define this inside the scope of init (As it only required there)? (or maybe we can use GLOBALS.can_edit directly?)